SemanticSearch.md

```diff
@@ -0,0 +1,299 @@
+            "id": "99-9999.99",
+            "weight": 1.0
+          }
+        ],
+        "related_keywords": [
+          {
+            "name": "extracted keyword related keyword",
+            "weight": 1.0
+          }
+        ],
+        "related_keywords_recruiter": [
+          {
+            "name": "extracted keyword related keyword",
+            "weight": 1.0
+          }
+        ],
+        "related_keywords_jobseeker": [
+          {
+            "name": "extracted keyword related keyword",
+            "weight": 1.0
+          }
+        ],
+        "skills": [
+          {
+            "name": "extracted keyword skill",
+            "weight": 1.0
+          }
+        ],
+        "raw_job_titles_recruiter": [
+          {
+            "name": "extracted keyword job title",
+            "weight":  1.0
+          }
+        ],
+        "raw_job_titles_jobseeker": [
+          {
+            "name": "extracted keyword job title",
+            "weight":  1.0
+          }
+        ],
+        "job_level": [
+         {
+            "name": "extracted keyword job level",
+            "weight": 1.0
+         }
+        ],
+        "text_kernel_related_keywords": [
+         {
+            "name": "extracted keyword Text Kernel related keyword"
+            "weight": 1.0
+         }
+       ]
+      }
+    }
+  ],
+  "job_titles": [
+    {
+      "name": "some title",
+      "id": "1",
+      "weight": 1.0
+    }
+  ],
+  "occupations": [
+    {
+      "name": "some occupation",
+      "id": "99-9999.00",
+      "weight": 1.0
+    }
+  ],
+  "related_keywords": [
+    {
+      "name": "some keyword",
+      "weight": 1.0
+    }
+  ],
+  "related_keywords_recruiter": [
+    {
+      "name": "some keyword",
+      "weight": 1.0
+    }
+  ],
+  "related_keywords_jobseeker": [
+    {
+      "name": "some keyword",
+      "weight": 1.0
+    }
+  ],
+  "skills": [
+    {
+      "name": "some skill",
+      "weight": 1.0
+    }
+  ],
+  "raw_job_titles_recruiter": [
+    {
+      "name": "some title",
+      "weight": 1.0
+    }
+  ],
+  "raw_job_titles_jobseeker": [
+    {
+      "name": "some title",
+      "weight": 1.0
+    }
+  ],
+  "job_level": [
+    {
+      "name": "some job level",
+      "weight": 1.0
+    }
+  ],
+  "text_kernel_related_keywords": [
+    {
+      "name": "some Text Kernel related keyword"
+      "weight": 1.0
+    }
+  ],
+  "versions": {
+    "job_titles": "CaroteneV3",
+    "occupations": "ONet17",
+    "skills": "SkillsV4",
+    "interesting_keywords": "InterestingTermsV3",
+    "related_keywords": "RelatedSearchTermsV1",
+    "related_keywords_recruiter": "RelatedSearchTermsRecruiterV1",
+    "related_keywords_jobseeker": "RelatedSearchTermsJobSeekerV1",
+    "raw_job_titles_jobseeker": "RawJobTitlesJobSeekerV1",
+    "raw_job_titles_recruiter": "RawJobTitlesRecruiterV1"
+    "job_level": "JobLevel"
+    "text_kernel_related_keywords": "TextKernelRelatedSearchTerms"
+  }
+}
+```
+
```

Comment:
Should we maybe describe the difference between the extracted keywords enrichments and the summary enrichments?